package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "install-ispc-action",
+  "version": "1.0.0",
+  "description": "Install ISPC GitHub action.",
+  "main": "src/main.js",
+  "type": "module",
+  "scripts": {
+    "run": "node src/main.js",
+    "build": "esbuild --bundle src/main.js --outfile=build/main.cjs --platform=node --format=cjs"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ScatteredRay/install-ispc-action.git"
+  },
+  "author": "Indy Ray",
+  "license": "MIT",
```

Your repo has Apache 2.0 license, so I assume it should be consistent everywhere.
```suggestion
  "license": "Apache-2.0",
```